tendrl-apid.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Tendrl Api Daemon
+
+[Service]
+Type=simple
+ExecStart=vendor/bin/puma -e development -d
+ExecReload=/bin/kill -HUP $MAINPID
```

Comment:
Better read the puma documentation to find out whether it can handle SIGHUP or not. It shut down when I tried it once, but I haven't checked the documentation to see what the expected behaviour is.